### PR TITLE
cleanup console backend logging

### PR DESCRIPTION
**Pull request by jamestalton**
Signed-off-by: James Talton <jtalton@redhat.com>

### CODE REVIEW

File: backend/src/lib/server.ts [hunk 1]
```diff
@@ -96,23 +96,20 @@ export function startServer(options: ServerOptions): Promise<Http2Server | undef
                         if (options.logRequest) {
                             options.logRequest(req, res)
                         } else {
+                            if (req.url === '/authenticated') return
                             if (req.url === '/readinessProbe') return
                             if (req.url === '/livenessProbe') return
+                            if (req.url === '/apis/authorization.k8s.io/v1/selfsubjectaccessreviews') return
```
Comment (KevinFCormier) [inline backend/src/lib/server.ts:102]: Is this related to logging?

Comment (jamestalton) [reply backend/src/lib/server.ts:102]: Yes, those lines cause a lot of logging and it makes it harder to tell what real actions happened on the backend.
It seems like those could be debugged easier from the frontend dev tools networking console tab easier.
